core/PRP/Physics/plGenericPhysical.cpp

```diff
@@ -816,4 +816,6 @@ void plGenericPhysical::calcBoxBounds(size_t numPoints, const hsVector3* points)
     hsBounds3Corners corners = bounds.getCorners();
     fVerts.reserve(corners.size());
     std::copy(corners.begin(), corners.end(), std::back_inserter(fVerts));
+    fVerts.reserve(std::size(hsBounds3::CornerIndices));
```

Comment:
`std::size` is C++17...  You could convert the `CornerIndices` to a `std::array` though to get around the need for it.

Reply:
Ah, that's what I was just wondering about. Thanks for the info. Give me a second...

Don't hesitate to nitpick if it's still suboptimal, my C++ isn't great.